documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -5,80 +5,104 @@
 = Renewing your own CA certificates
 
-This procedure describes how to renew CA certificates and private keys that you previously installed.
-You will need to follow this procedure during the renewal period in order to replace CA certificates which will soon expire.
+This procedure describes how to renew CA certificates and keys that you previously installed.
+You need to follow this procedure during the renewal period in order to replace CA certificates which will soon expire.
+
+You can use this procedure to install your own cluster or client CA certificates.
+
+The procedure describes renewal of CA certificates in PEM format.
+You can also use certificates in PKCS #12 format.
 
 .Prerequisites
 
 * The Cluster Operator is running.
 * A Kafka cluster in which you previously installed your own CA certificates and private keys.
-* New cluster and clients X.509 certificates and keys in PEM format. These could be generated using `openssl` using a command such as:
-+
+* New cluster and clients X.509 certificates and keys in PEM format.
+
+These could be generated using `openssl` using a command such as:
+
 [source,shell,subs="+quotes"]
-openssl req -x509 -new -days _<validity>_ --nodes -out ca.crt -keyout ca.key
+openssl req -x509 -new -days _NUMBER-OF-DAYS-VALID_ --nodes -out ca.crt -keyout ca.key
 
 .Procedure
 
-. Establish what CA certificates already exist in the `Secret`:
-+
-Use the following commands:
+. Check the details of the current CA certificates in the `Secret`:
 +
 [source,shell,subs="+quotes"]
-kubectl describe secret _<ca-cert-secret>_
+kubectl describe secret _CA-CERTIFICATE-SECRET_
 +
+_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
 
-. Prepare a directory containing the existing CA certificates in the secret.
+. Create a directory to contain the existing CA certificates in the secret.
 +
 [source,shell,subs="+quotes"]
 ----
 mkdir new-ca-cert-secret
 cd new-ca-cert-secret
 ----
-For each certificate _<ca-certificate>_ from the previous step, run:
+
+. Fetch the secret for each CA certificate you wish to renew:
 +
 [source,shell,subs="+quotes"]
-----
-# Fetch the existing secret
-kubectl get secret _<ca-cert-secret>_ -o 'jsonpath={.data._<ca-certificate>_}' | base64 -d > _<ca-certificate>_
-----
+kubectl get secret _CA-CERTIFICATE-SECRET_ -o 'jsonpath={.data._CA-CERTIFICATE_}' | base64 -d > _CA-CERTIFICATE_
++
+Replace _CA-CERTIFICATE_ with the name of each CA certificate.
 
-. Rename the old `ca.crt` file to `ca_<date>_.crt`, where _<date>_ is the certificate expiry date in the format _<year>_-_<month>_-_<day>_T_<hour>_-_<minute>_-_<second>_Z, for example `ca-2018-09-27T17-32-00Z.crt`. 
+. Rename the old `ca.crt` file as `ca-__DATE__.crt`,
+where _DATE_ is the certificate expiry date in the format _YEAR-MONTH-DAYTHOUR-MINUTE-SECONDZ_.
```

Comment:
Missing underscore - `_Z`?

Reply:
No. That's the format: yyyy-mm-dd**T**hh-mm-ss**Z**

Reply:
We really need to highlight the T and Z in there. `_YEAR-MONTH-DAYTHOUR-MINUTE-SECONDZ_` is doesn't really do it.